eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -376,18 +379,27 @@ function CheckDependencies()
         }
     }
 
+    # helm version example: v3.11.2+g912ebc1
+    $helmVersionString = (helm version --short).substring(1) -replace '\+.*',''
+    $helmVersion = [AzureEngSemanticVersion]::new($helmVersionString)
+    $minHelmVersion = [AzureEngSemanticVersion]::new($MIN_HELM_VERSION)
+    if ($helmVersion.CompareTo($minHelmVersion) -lt 0) {
+        throw "Please update helm to version >= $MIN_HELM_VERSION (current version: $helmVersionString)`nAdditional information for updating helm version can be found here: https://helm.sh/docs/intro/install/"
+    }
+
     if ($shouldError) {
         exit 1
     }
 
 }
 
 function generateRetryTestsHelmValues ($pkg, $releaseName, $generatedHelmValues) {
+    CheckDependencies
```

Comment:
This gets run on entry, so we don't need it here: https://github.com/Azure/azure-sdk-tools/blob/7554cbcd100d828bb9ba75cd4993551199660f76/eng/common/scripts/stress-testing/deploy-stress-tests.ps1#L41

```suggestion
```